Resources/Private/Language/locallang.xlf

```diff
@@ -100,6 +100,14 @@
         <source>You have successfully changed your password and have been logged in automatically.</source>
       </trans-unit>
 
+      <trans-unit id="updatePassword.completed">
+        <source>You have successfully updated your password and may login with your new password next time.</source>
+      </trans-unit>
+
+      <trans-unit id="validationErrors.errors">
```

Comment:
You can drop `resetPassword.errors` now.

Reply:
This translation key is used in template ShowPasswordResetForm.

Reply:
Yes, please unify this. :-)